examples/agents/rewoo/README.md

```
@@ -120,85 +120,95 @@ The ReWOO agent is configured through the `config.yml` file. The following confi
 Run the following command from the root of the NeMo Agent toolkit repo to execute this workflow with the specified input:
 
 ```bash
-nat run --config_file=examples/agents/rewoo/configs/config.yml --input "Which city held the Olympic game in the year represented by the bigger number of 1996 and 2004?"
+nat run --config_file=examples/agents/rewoo/configs/config.yml --input "Make a joke comparing Elon and Zuckburg's birthdays?"
 ```
 
 **Expected Workflow Output**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)